src/tasks/AndroidAppBuilder/AndroidAppBuilder.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>$(NetCoreAppToolCurrent)</TargetFramework>
+    <TargetFramework>$(TargetFrameworkForNetCore)</TargetFramework>
```

Comment:
Is this intended just for tasks? If so, then we should name it accordingly. It's not clear from the name, what it is for, or when to use it.

Reply:
Perhaps, but it looks quite consistent in other places, e.g. in MonoAOTCompiler task `<TargetFrameworks>$(TargetFrameworkForNetCore);$(TargetFrameworkForNETFramework)</TargetFrameworks>`

Reply:
It seems like the property is intended just to be used in `src/tasks`. Not sure if `TargetFrameworkForNETFramework` is also intended to be just used in `src/tasks` as well, but if it is perhaps we can rename them both to be something like `TargetFrameworkForNETCoreTasks` and `TargetFrameworkForNETFrameworkTasks`?

Reply:
@radical if it is intended for just the tasks, do you think the above would be proper, or did you have something else in mind?

Reply:
> .. but if it is perhaps we can rename them both to be something like TargetFrameworkForNETCoreTasks and TargetFrameworkForNETFrameworkTasks

Sounds good.

Reply:
A quick search showed that the `TargetFrameworkForNETFramework` property is used within `src\task` only so I've made the following change:
`TargetFrameworkForNETFramework` -> `TargetFrameworkForNETFrameworkTasks`  
`TargetFrameworkForNetCore` -> `TargetFrameworkForNETCoreTasks`